spec/models/guest_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Guest, type: :model do
+  describe "relationships" do
+    it { should have_many :rooms }
+  end
+
+  describe "validations" do
+    it { should validate_presence_of(:name) }
```

Comment:
Nice work including these validations! But just keep in mind that you're not required to do validations on an IC because of the time restraint. We're more concerned about your testing the model relationships on an IC. You can skip validations entirely if you want (but we do want you to use them on projects!)